lib/addons/cloudwatch-logs/index.ts

```diff
@@ -109,6 +109,6 @@ function populateValues(clusterInfo: ClusterInfo, helmOptions: CloudWatchLogsAdd
     setPath(values, "cloudWatchLogs.logGroupTemplate", `${helmOptions.logGroupPrefix}/$kubernetes['namespace_name']`);
     setPath(values, "cloudWatchLogs.logStreamTemplate", "$kubernetes['container_name'].$kubernetes['pod_name']");
     setPath(values, "cloudWatchLogs.log_key", "log");
```

Comment:
since we are fixing the chart values, do you mind also fixing log_key here which is also incorrectly using snake case - mustbe `logKey` based on https://github.com/aws/eks-charts/blob/master/stable/aws-for-fluent-bit/values.yaml.

Reply:
Ok, make sense, fixed in my last commit